config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -526,6 +526,80 @@ periodics:
           memory: 9Gi
           cpu: 7
 
+- interval: 4h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-dependencies
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: kind-master-dependencies
+    description: Runs tests after updating dependencies to latest versions
+    testgrid-alert-email: davanum@gmail.com
+    testgrid-num-columns-recent: '6'
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/krte:v20250513-98d205aae3-master
+      command:
+      - wrapper.sh
+      - bash
+      - -c
+      - |
```

Comment:
we should move this into a file, the job is cloning test-infra already anyhow

it's hard to debug locally and hard to iterate on safely when it's all inline

Reply:
@BenTheElder yep! will do in the next round of updates

Reply:
this also shouldnt be in kubernetes-kind.yaml

Reply:
Understood